### PR TITLE
EZP-28792: Change password

### PR DESCRIPTION
**Pull request by DominikaK**
https://jira.ez.no/browse/EZP-28792

(2.1)

At the same time adds a new "User management" page which is sure to come useful in future.

### CODE REVIEW

**Comment by DominikaK** [comment]
ping @mikadamczyk for review

File: docs/guide/user_management.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# User management
+
+## Passwords
+
+### Changing passwords
+
+The user may request to change their password, of may forget it and ask to have it reset.
```
Comment (MagdalenaZuba) [inline docs/guide/user_management.md]: or may

File: docs/guide/user_management.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+To change their password, the user must have the `user/password` permission.
```
Comment (MagdalenaZuba) [inline docs/guide/user_management.md]: To change password, / To do it (as continuation of previous sentence)

File: docs/guide/user_management.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+In such a case an email is sent to the user with a one-time token. It that allows them to create a new password.
```
Comment (MagdalenaZuba) [inline docs/guide/user_management.md]: it or that?

**Pull request merged**